tools/serverpod_cli/lib/src/internal_tools/analyze_pubspecs.dart

```diff
@@ -62,8 +81,23 @@ Future<void> _checkLatestVersion(
         log.error(e.message);
       }
 
-      if (latestPubVersion != null &&
-          depVersion != '^${latestPubVersion.toString()}') {
+      if (ignoreServerpodPackages && depName.startsWith('serverpod')) {
```

Comment:
Could this be an issue for potential community packages (i.e. `serverpod-somepackage-adapter`)?  If yes, maybe we need to have an array of known serverpod packages to check against instead.

Reply:
Yes that is correct, if a package is named something like that we would ignore it.

But we would only analyze it if one our our Serverpod packages would depend on it. 

This command is only an internal tool to manager our mono repo. So we can probably ignore the wider community use.

Reply:
This is a good observation. Perhaps add a todo and a link to an issue, so it's easy to fix if we run into it in the future?

Reply:
Sounds like a plan! 👍